client/src/components/body/EducatorResources/components/ResourceRow.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import PropTypes from "prop-types";
+import ALLButton from "../../../all-components/ALLButton";
+import ComingSoon from "../../../../assets/images/ComingSoonVideo.svg";
+import { COMING_SOON } from "../../../../constants/educatorResources/LabOverview";
+
+const ResourceRow = (props) => {
+  const { id, title, subTitle, description, image, walkthroughVideo, slides } =
+    props;
+
+  return (
+    <li key={id} className={"tw-my-5"}>
+      <div className={"tw-flex xs:tw-flex-col md:tw-flex-row"}>
+        <div
+          className={"tw-flex tw-flex-col xs:tw-w-full md:tw-w-1/2 tw-gap-3"}
+        >
+          <p className={"tw-font-calibri tw-font-extrabold tw-text-xl"}>
+            {id ? `Lab ${id}:` : ""} {title}
+          </p>
+          <p className="xs:tw-text-xs md:tw-text-sm tw-text-justify">
+            <hr />
+            {description}
+          </p>
+          {slides && !slides.includes(COMING_SOON) && (
+            <a href={slides} download>
+              <ALLButton label={`Download ${subTitle} Slideshow`} />
```

Comment:
I wonder if it would be better to just say `Download Slideshow`. We are already referencing the lab name. I don't we need to again in the button.

![image](https://github.com/user-attachments/assets/7cf2e4b6-657f-493d-a1df-9a37b3d65d13)

Reply:
You're right, too long and makes the buttons look like different sizes